pipelines/build/common/config_regeneration.groovy

```diff
@@ -47,6 +48,7 @@ class Regeneration implements Serializable {
         String javaVersion,
         Map<String, Map<String, ?>> buildConfigurations,
         Map<String, ?> targetConfigurations,
+        String activeNodeTimeout,
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs

Reply:
Sorted.